src/rustdoc_cmd.rs

```diff
@@ -129,57 +129,48 @@ impl RustdocCommand {
         if !self.deps {
             cmd.arg("--no-deps");
         }
-        if config.is_stderr_tty() {
-            cmd.arg("--color=always");
-        }
+
+        // TODO
```

Comment:
We used to only add the `--color=always` flag if `stderr` was a tty, but we can respect the user's color choice more now *if we want*.  I can also easily revert to the earlier behavior.